### ISSUE TITLE
Issue with the update_gie_alsi_archive function

### ISSUE THREAD

**Issue by AdibouD** [ec-jrc/eurogastp#1]
Hello,

I hope I'm not mistaken, but the call to the _update_gie_alsi_archive_ function seems to be encountering errors. I did not have this issue yesterday when downloading the various xlsx files. I'm not sure if there has been a change in the ALSI database in this regard.
I have no issues with the one for the AGSI database.
```
Traceback (most recent call last):

  Cell In[16], line 1
    gtp.update_gie_alsi_archive(download_start, download_end, proxy=proxy,

  File ~\anaconda3\Lib\site-packages\eurogastp\gie.py:369 in update_gie_alsi_archive
    df_new = download_gie_alsi(start_date, end_date, api_key=api_key,

  File ~\anaconda3\Lib\site-packages\eurogastp\gie.py:115 in download_gie_alsi
    df[float_cols] = df[float_cols].replace('-', '0').astype(float)

  File ~\anaconda3\Lib\site-packages\pandas\core\generic.py:6534 in astype
    new_data = self._mgr.astype(dtype=dtype, copy=copy, errors=errors)

  File ~\anaconda3\Lib\site-packages\pandas\core\internals\managers.py:414 in astype
    return self.apply(

  File ~\anaconda3\Lib\site-packages\pandas\core\internals\managers.py:354 in apply
    applied = getattr(b, f)(**kwargs)

  File ~\anaconda3\Lib\site-packages\pandas\core\internals\blocks.py:616 in astype
    new_values = astype_array_safe(values, dtype, copy=copy, errors=errors)

  File ~\anaconda3\Lib\site-packages\pandas\core\dtypes\astype.py:238 in astype_array_safe
    new_values = astype_array(values, dtype, copy=copy)

  File ~\anaconda3\Lib\site-packages\pandas\core\dtypes\astype.py:183 in astype_array
    values = _astype_nansafe(values, dtype, copy=copy)

  File ~\anaconda3\Lib\site-packages\pandas\core\dtypes\astype.py:134 in _astype_nansafe
    return arr.astype(dtype, copy=True)

TypeError: float() argument must be a string or a real number, not 'dict'
```

**Comment by proggy** [comment]
I can confirm this issue (under Python 3.11.4, Windows). I'm working on a fix.

**Comment by proggy** [comment]
fixed in v0.1.3
thanks for reporting the issue
